src/styles/App.scss

```diff
@@ -47,19 +47,3 @@ body {
 a.catalog-image-link {
   text-decoration: none;
 }
-
-// Needed until the following PR gets resolved
-// https://github.com/material-components/material-components-web/pull/2704
-@media (max-width: 599px) {
-  .mdc-top-app-bar--fixed-adjust {
-    margin-top: 0;
-    padding-top: 56px;
-  }
-}
-
-// Needed until MDC Web #2702 is resolved
```

Comment:
Doesn't this cause a regression unless we also actually update the card JSX to use mdc-button-icon? AFAICT, removing these styles resurfaces the original bug, where the ripples on the 2nd and 3rd icon are the wrong size when you hover or focus.

Reply:
`icon-toggle` -> `icon-button` is in #161